File: www/kd/cgi-codes/exit.py
from __future__ import print_function

import sys
import os

import cgi
import cgitb

form = cgi.FieldStorage()

code = form.getvalue("code", "0")

print("Content-Type: text/plain", end="\r\n\r\n");
print("PYTHON : will exit (" + code + ")\r\n");

exit (int(code));

File: www/kd/cgi-codes/exit.pl
#!/usr/bin/perl

use strict;
use warnings;
use CGI;

my $cgi = CGI->new;

my $code = $cgi->param('code');

print("Content-Type: text/plain\r\n\r\n");
print("PERL : will exit (", $code, ")\r\n");
exit ($code);

